src/app/utils/tests/TestUtils.spec.ts

```diff
@@ -454,21 +453,19 @@ export class TestUtils {
 export async function setupEmulators(): Promise<unknown> {
     await TestBed.configureTestingModule({
         imports: [
-            AngularFirestoreModule,
             HttpClientModule,
-            AngularFireModule.initializeApp(environment.firebaseConfig),
+            FirebaseProviders.app(),
+            FirebaseProviders.firestore(),
+            FirebaseProviders.auth(),
         ],
         providers: [
-            { provide: USE_AUTH_EMULATOR, useValue: environment.emulatorConfig.auth },
-            { provide: USE_DATABASE_EMULATOR, useValue: environment.emulatorConfig.database },
-            { provide: USE_FIRESTORE_EMULATOR, useValue: environment.emulatorConfig.firestore },
-            { provide: USE_FUNCTIONS_EMULATOR, useValue: environment.emulatorConfig.functions },
             AuthenticationService,
         ],
-        schemas: [CUSTOM_ELEMENTS_SCHEMA],
     }).compileComponents();
+    TestBed.inject(Firestore.Firestore);
+    TestBed.inject(Auth.Auth);
     const http: HttpClient = TestBed.inject(HttpClient);
-    // Clear the firestore data before each test
+    // Clear the content of the firestore database
```

Comment:
noooooooooon :D (the test of the mock db, RIGHT :D)

Reply:
oui tout à fait, clarifié !